meta-dts-distro/recipes-bsp/coreboot-utils/ectool_git.bb

```diff
@@ -2,6 +2,8 @@ require coreboot-utils.inc
 
 SUMMARY = "Dumps the RAM of a laptop's Embedded/Environmental Controller (EC)."
 
+PV = "1.0+git${SRCPV}"
```

Comment:
Can you maybe put this in `coreboot-utils.inc` that is included? If yes then you could only change that one file instead of ~10.

Reply:
Or maybe not as each tool might have their own version:

```shell
bash-5.2# ectool -v
ectool v0.1 -- Copyright (C) 2008-2009 coresystems GmbH
```

Reply:
Our coreboot fork is tagged like this: `protectli_vault_glk_v1.1.0`. I don't know if using such tags in `PV` makes sense, what do you think?

Reply:
> Our coreboot fork is tagged like this: protectli_vault_glk_v1.1.0. I don't know if using such tags in PV makes sense, what do you think?

When suggesting this, I had in mind rather the coreboto release tag, not our dasharo release tags: https://doc.coreboot.org/releases/index.html

Reply:
@PLangowski I'm not too sure, I think each platform has each own branch/tag and versioning.
Newest tag containing current revision is `protectli_vault_jsl_v0.9.2` or `protectli_vault_jsl_v0.9.3-rc1` if you want to use release candidates too. And `protectli_vault_glk_v1.1.0` doesn't contain revision used on develop from what I can see. Maybe ask someone from Dasharo team if there is some kind of common versioning before they use platform specific ones? I guess it could be 4.21 (from branch name)?

Reply:
If we still use 4.21 branch, we should use 4.21, as this is the version we base on.

But we already have version 24.02 on main `dasharo` branch right now as well.

Reply:
68f9926e5e6a51d61325ef1f0cd8e6a3cba7c8e5